docs/backtesting.md

```diff
@@ -157,16 +157,27 @@ A backtesting result will look like that:
 | ADA/BTC  |      1 |           0.89 |           0.89 |       0.00004434 |           0.44 | 6:00:00        |     1 |      0 |       0 |
 | LTC/BTC  |      1 |           0.68 |           0.68 |       0.00003421 |           0.34 | 2:00:00        |     1 |      0 |       0 |
 | TOTAL    |      2 |           0.78 |           1.57 |       0.00007855 |           0.78 | 4:00:00        |     2 |      0 |       0 |
+============ SUMMARY METRICS =============
+| Metric           | Value               |
+|------------------+---------------------|
+| Total trades     | 429                 |
+| First trade      | 2019-01-01 18:30:00 |
+| First trade Pair | EOS/USDT            |
+| Backtesting from | 2019-01-01 00:00:00 |
+| Backtesting to   | 2019-05-01 00:00:00 |
+| Trades per day   | 3.575               |
+|                  |                     |
```

Comment:
do we really need the empty line here?

Reply:
technically - absolutely not (and i guess you know that, too 😆 )
my initial idea was to split it based on "base data" and "additional stuff" - to improve readability.

If you think it's "readable enough" without it, we can remove it - no problem